starboard/android/shared/drm_system.cc

```diff
@@ -206,10 +211,33 @@ DrmSystem::DrmSystem(
     return;
   }
   j_media_crypto_ = env->ConvertLocalRefToGlobalRef(j_media_crypto_);
+
+  Start();
+}
+
+void DrmSystem::Run() {
+  JniEnvExt* env = JniEnvExt::Get();
+  bool result = env->CallBooleanMethodOrAbort(
+      j_media_drm_bridge_, "createMediaCryptoSession", "()Z");
```

Comment:
Can |j_media_drm_bridge_| be deleted in the dtor (on the main thread) before using it?

Reply:
In theory it could, perhaps in some edge case such as unsuccessful playback start. Added a join in the dtor to ensure it finishes first.